### ISSUE TITLE
++  mink jet mismatch: %1 block autocons

### ISSUE THREAD

**Issue by ohAitch** [urbit/urbit#357]
++mink, as written, supports having two blocking requests in a cell, e.g.

```
> (mink 0^[[11 1 20] [11 1 21]] ,~)
::  ought to be
[%1 ~[20 21]]
```

This is not reflected in u3's idea of nock execution semantics(actual result `[%1 ~[20]]`), which causes causing e.g. a `(turn paths file)` to induce O(`(lent paths)`) network round-trips in ford instead of all the blocks being computed in one pass.

**Comment by ohAitch** [comment]
@philipcmonk which, apparently, it doesn't at the C level P:

**Comment by ohAitch** [comment]
Should this just be patched in `++mock` with a comment documenting the desired future behavior?

**Comment by jtobin** [comment]
@joemfb Was this one resolved by #2285 afterwards?

**Comment by joemfb** [comment]
There's a fix on a branch (`na-release/next-vere`), but it'll require a breach to release it.

**Comment by joemfb** [comment]
Fixed in #2285, finally released.
